Corona-Warn-App/src/main/res/layout/analytics_ppa_userinfo_input_fragment.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <androidx.appcompat.widget.Toolbar
+            android:id="@+id/toolbar"
+            style="@style/CWAToolbar.Close"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/colorBackground"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:title="Ihr Alter/Bundesland/Kreis " />
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/input_list"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
```

Comment:
Please use `tools:listitem` ? it gives the other overview if later the layout is visited